Pipfile

```diff
@@ -6,10 +6,10 @@ verify_ssl = true
 [dev-packages]
 apache-airflow = "==2.9.1"
 pylint = "*"
-pytest = "==8.2.0"
+pytest = "==7.4.4"
 moto = "==5.0.6"
-requests-mock = "==1.11.0"
-boto3 = "==1.33.13"
+requests-mock = "==1.12.1"
+boto3 = "==1.34.69"
```

Comment:
Just a little confused, request-mock and boto3 are upgrading but the title of this PR says you are downgrading.

Reply:
Should probably just be titled "updates" -- additional changes after I pushed the original up to keep everything in one PR. Some are up, some are down. These changes are the intended changes per the constraints file.

Reply:
Fixed title